docker-compose.e2e-basic.yml

```diff
@@ -0,0 +1,201 @@
+version: '3.0'
+services:
+  morqa:
+    image: mysteriumnetwork/quality-oracle:latest
```

Comment:
Not sure if about this

Reply:
When we use latest images of our services e2e tests will be able to catch possible issues faster regarding backwards compatibility.